python/lsst/base/packages.py

```diff
@@ -512,6 +524,17 @@ def difference(self, other):
         return {pkg: (self._packages[pkg], other._packages[pkg]) for
                 pkg in self._names & other._names if self._packages[pkg] != other._packages[pkg]}
 
+    def to_dict(self):
```

Comment:
No particular objection to this, but if the whole thing _is_ a `dict`, then this should be unnecessary.

Reply:
Removed.